L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorSuperTriggerCellImpl.h

```diff
@@ -23,15 +26,15 @@ class HGCalConcentratorSuperTriggerCellImpl
   private:
 
     int getSuperTriggerCellId(int detid) const ;
-    static const int kSplit12_ = 0x3a;
-    static const int kSplit3_ = 0x30;
+    static std::map<int,int> kSplit_;
```

Comment:
Should be `static const`

Reply:
it doesn't compile with `static const` - any ideas?

Reply:
The map accessor `[]` (which is non const) should be replaced with `at()`

Reply:
Thanks